rules/windows/process_creation/proc_creation_win_renamed_plink.yml

```diff
@@ -0,0 +1,23 @@
+title: Execution Of Renamed Plink Binary
+id: 1c12727d-02bf-45ff-a9f3-d49806a3cf43
+status: experimental
+description: Execution of a renamed version of the Plink binary
+author: Nasreddine Bencherchali
+references:
+    - https://thedfirreport.com/2022/06/06/will-the-real-msiexec-please-stand-up-exploit-leads-to-data-exfiltration/
```

Comment:
What is Plink  `- https://the.earth.li/~sgtatham/putty/0.58/htmldoc/Chapter7.html`

Reply:
Fixed that one